.github/workflows/build.yml

```diff
@@ -13,8 +13,10 @@ jobs:
       - uses: actions/setup-node@v4
         with:
           node-version: 22
+      - name: Enable Corepack
+        run: corepack enable
```

Comment:
The install process for latest versions of Yarn
https://yarnpkg.com/migration/guide#migration-steps